osdk/src/cli.rs

```diff
@@ -264,6 +264,12 @@ pub struct CommonArgs {
         global = true
     )]
     pub linux_x86_legacy_boot: bool,
+    #[arg(
+        long = "strip-elf",
+        help = "Strip the built kernel ELF file for a smaller size",
+        global = true
```

Comment:
What is the default value in `OSDK.toml`? This is the default value for the CLI, not the configuration file, right?

Reply:
For `OSDK.toml` it defaults to false. In CLI the default is false (not set). If set this is true.